src/main/java/com/commercetools/project/sync/model/ProductSyncCustomRequest.java

```diff
@@ -0,0 +1,48 @@
+package com.commercetools.project.sync.model;
+
+import static java.lang.String.format;
+
+import com.commercetools.project.sync.exception.CliException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+
+public final class ProductSyncCustomRequest {
+
+  private Long fetchSize;
```

Comment:
How this is working? I guess we need to update the test cases and use limit and where as field names?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/243/commits/38a9dc004c1e7cf50505480bfb6a95e17574a03c